sonar-java-plugin/src/main/resources/static/documentation.md

```diff
@@ -147,11 +147,18 @@ The tutorial [Writing Custom Java Rules 101](https://redirect.sonarsource.com/do
 All the API changes are related to ECJ utility methods that were commonly used in the analyzer and could benefit the implementation of 
 custom rules.
 
-* New Method `Symbol.MethodSymbol.isOverridable()`. Returns whether this method is overridable.
-* New Method `Symbol.MethodSymbol.isVarArgsMethod()`. Returns whether this method has a vararg parameter.
-* New Method `Symbol.MethodSymbol.isDefaultMethod()`. Returns whether this method has a default implementation.
-* New Method `Symbol.MethodSymbol.isParametrizedMethod()`. Returns whether this method has type parameters.
-* New Method `Symbol.MethodSymbol.isSynchronizedMethod()`. Returns whether this method is synchronized.
+* New method: `Symbol.TypeSymbol#superTypes()`. Returns the list of super types of this type.
```

Comment:
Consistency: `#` or `.`? Don't mix. And I think it should be `.`, `#` is used only in the JavaDoc links.